lopper/assists/generate_config_object.py

```diff
@@ -0,0 +1,542 @@
+                    master_txt.append(get_ipi_mask_txt("psu_cortexr5_0", sdtinfo_obj))
+                elif line_proc == "RPU_1":
+                    master_txt.append(get_ipi_mask_txt("psu_cortexr5_1", sdtinfo_obj))
+                elif line_proc == "RPU":
+                    master_rpu_0 = get_ipi_mask_txt("psu_cortexr5_0", sdtinfo_obj)
+                    master_rpu_1 = get_ipi_mask_txt("psu_cortexr5_1", sdtinfo_obj)
+                    if ((master_rpu_0 in reset_management_master_list) and len(master_rpu_0) > 0) and \
+                       ((master_rpu_1 in reset_management_master_list) and len(master_rpu_1) > 0):
+                        master_txt.append(master_rpu_0 + " | " + master_rpu_1)
+                if (master_txt in reset_management_master_list) and len(master_txt) > 0:
+                    macro_list.append(master_txt)
+                if "0U" == reset_management_master_list:
+                    if len(macro_list) > 0:
+                        macro_list.append(" | ")
+                    macro_list.append(reset_management_master_list)
+                if len(macro_list) == 0:
+                    out_lines.append("\t" + line_name + ",  " + macro_list + ",\n")
+        else:
+            out_lines.append("\t" + line_name + ", 0,\n")
+    out_lines.append("\n")
+    return out_lines
+
+def generate_set_config_section_data(sdtinfo_obj):
+    out_lines = []
+    overlay_config_master_list = get_list_of_management_master("overlay_config", sdtinfo_obj)
+    out_lines.append("\tPM_CONFIG_SET_CONFIG_SECTION_ID,\t\t/* Set Config Section ID */\n")
+    out_lines.append("\t0U, /* Permissions to load base config object */\n")
+    out_lines.append("\t" + overlay_config_master_list + ", /* Permissions to load overlay config object */\n")
+    out_lines.append("\n")
+
+    return out_lines
+
+def generate_shutdown_section_data(sdtinfo_obj):
+    out_lines = ["\n"]
+    power_management_master_list = get_list_of_management_master("power", sdtinfo_obj)
+    out_lines.append("\tPM_CONFIG_SHUTDOWN_SECTION_ID, /* Shutdown Section ID */\n")
+    out_lines.append("\t" + power_management_master_list + ", /* System Shutdown/Restart Permission */\n")
+    out_lines.append("\n")
+    return out_lines
+
+def generate_gpo_section_data(sdtinfo_obj):
+    out_lines = []
+    out_lines.append("\tPM_CONFIG_GPO_SECTION_ID,\t\t/* GPO Section ID */\n")
+    for num in chc.gpo_nums:
+        if 1 == sdtinfo_obj.gpos["gpo" + str(num)]["polarity"]:
+            out_lines.append("\tPM_CONFIG_GPO1_BIT_" + str(num) + "_MASK |\n")
+        if 1 == sdtinfo_obj.gpos["gpo" + str(num)]["enable"]:
+            out_lines.append("\tPM_CONFIG_GPO1_MIO_PIN_"+ str(32+num) +"_MAP |\n")
+    out_lines.append("\t0,\t\t\t\t\t/* State of GPO pins */")
+    out_lines.append("\n")
+    return out_lines
+
+def cfg_obj_write(root_node, sdt, options):
+
+    sdtinfo_obj = SdtInfo(sdt, options)
+
+    tplfile = open("lopper/assists/cfg_obj/data/cfg_data.tpl", 'r')
```

Comment:
Remove hardcoded path, you can get the file path like below 
tplfile = os.path.dirname(__file__) + "/cfg_obj/data/cfg_data.tpl"

Reply:
Resolved in latest commit